boards/arm/lpcxpresso54114_m0/pinmux.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2017,  NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <pinmux.h>
+#include <fsl_common.h>
+#include <fsl_lpc_iocon.h>
+
+#define IOCON_PIO_DIGITAL_EN	0x80u
```

Comment:
these defines should be somewhere common.

Reply:
Ok, fixed. This was not in MCUxpresso SDK device header file, but I have added the file pin_mux.h from SDK to ext. It defines these things.